src/System.Net.Sockets/src/System/Net/Sockets/BufferList.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System.Net.Sockets
```

Comment:
In case this file changed: please ensure that the public API supported in the 4.0.0.0 and 4.0.10.0 contracts is still available.